packages/backend/src/projectAdapters/dbtGitProjectAdapter.ts

```diff
@@ -44,8 +45,18 @@ const gitErrorHandler = (e: Error) => {
             { message: e.message },
         );
     }
+    if (e.message.includes('Repository not found')) {
```

Comment:
Specific error for repository not found errors from github, we should a more user friendly error message.